code/game/objects/items/devices/flashlight.dm

```diff
@@ -216,6 +216,9 @@
 	else
 		update_brightness()
 
+/obj/item/device/flashlight/flare/ground_act(var/obj/item/weapon/reagent_containers/RC)
```

Comment:
Nonstandard formatting, extract isn't used in these (and does in fact not apply to anything but food items since they are the only things for which the extract/process dichtomy matters)